docs/content/functioneel/informatieobjecten_api.md

```diff
@@ -0,0 +1,78 @@
+|----------------|----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| Aanleiding     | Een consumer wil (een lijst van) enkelvoudige informatieobjecten opvragen uit of een enkelvoudig informatieobject toevoegen aan de documentregistratiecomponent DRC.            |
+| *Toevoegen*    |      |
+| - voorwaarde   | Het enkelvoudg document dient beschikbaar te zijn voordat dit toegevoegd kan worden aan de DRC         |
+| - gevolg       | Na het succesvol toevoegen wordt een verwijzing in de vorm van een URI naar het enkelvoudig document teruggeven aan de comsumer.             |
+| Specificaties  | https://ref.tst.vng.cloud/drc/api/v1/schema/#operation/enkelvoudiginformatieobject_create   | 
+| *Wijzigen*     | Nog niet beschikbaar.     |
+| - voorwaarde   | Nog te bepalen.         |
+| - gevolg       | Nog te bepalen.             |
+| *Verwijderen*  | Nog niet beschikbaar.     |
+| - voorwaarde   | Nog te bepalen.         |
+| - gevolg       | Nog te bepalen.             |
+| *Raadplegen lijst*   |      |
+| - voorwaarde   | Geen voorwaarden.  |
+| - gevolg       | Een overzicht van in de Documentregistratiecomponent opgeslagen informatieobjecten wordt geretourneerd.|
+| Specificaties  | https://ref.tst.vng.cloud/drc/api/v1/schema/#operation/enkelvoudiginformatieobject_list   | 
+| *Raadplegen details*   |      |
+| - voorwaarde   | Bij het opvragen van een specifiek enkelvoudg informatieobject moet de identificatie van het informatieobject door de consumer aangeleverd worden.     |
+| - gevolg       | Een enkelvoudg informatieobject geïdentificeerd door de aangeleverde identificatie wordt geretourneerd. |
+| Gegevens       | Objecttype ENKELVOUDIGINFORMATIEOBJECT inclusief relatie naar INFORMATIEOBJECTTYPE.<br>Zie de volgende tabel voor de gegevens van deze resource                                 |
+| Specificaties  | https://ref.tst.vng.cloud/drc/api/v1/schema/#operation/enkelvoudiginformatieobject_read   | 
+| Samenhang      | - |
+| Bijzonderheden | Enkelvoudig Informatieobject is een subtype van Informatieobject. Beide objecttypen zijn "platgeslagen" zodat met het opvragen van een enkelvoudiginformatieobject ook de eigenschappen van het type informatieobject geretourneerd worden.
+|
+
+
+
+### Resource: Objectinformatieobjecten
+
+
+| **Aspect**     | **Beschrijving**                                                                                                                                                                 |
+|----------------|----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| Aanleiding     | Een consumer wil objectinformatieobjecten opvragen uit of een objectinformatieobject toevoegen aan de documentregistratiecomponent DRC.            |
```

Comment:
Is de aanleiding hier niet eerder iets als:

> Een consumer wil de relaties tussen informatieobjecten en andere objecten raadplegen

?

Reply:
Misschien kan het zelfs nog eenvoudiger: Een consumer wil weten bij welke objecten informatieobjecten horen.

Reply:
Ja, en ook (belangrijker voor dagdagelijks gebruik): "Een consumer wil weten welke informatieobjecten bij een object horen."

Reply:
Die laatste heb ik overgenomen, dat lijkt me inderdaad de meest voorkomende vraag.